frontend/src/components/EllipsisText.vue

```diff
@@ -1,17 +1,35 @@
 <template>
-  <span class="tooltip-wrapper w-full">
-    <span v-if="hasTextOverflow" class="tooltip whitespace-nowrap">
-      <slot></slot>
-    </span>
-    <div ref="textRef" class="truncate w-full"><slot></slot></div>
-  </span>
+  <NEllipsis
+    :line-clamp="lineClamp"
+    :tooltip="{
+      showArrow: false,
+      raw: true,
+      contentStyle: tooltipStyle,
+    }"
+  >
+    <slot></slot>
+  </NEllipsis>
 </template>
 
 <script lang="ts" setup>
-import { ref, computed } from "vue";
+import { NEllipsis } from "naive-ui";
```

Comment:
If this is a general component, then it should be moved into bbkit.

And for this particular case, can we just use tailwind css style? https://tailwindcss.com/docs/text-overflow

Reply:
> And for this particular case, can we just use tailwind css style?

The hardest part is not the ellipsis, the key is to have **both** ellipsis and a hover tooltip together, which is kind of tricky to implement.

I tried it before in https://github.com/bytebase/bytebase/pull/2262, but the effect is still far from my expectation. It doesn't perform well in table cells especially those in the middle places.

So I finally came up with the idea to encapsulate `NEllipsis ` with the style of our tooltip, making it both consistent in functionality and style.

Reply:
OK, defer to @LiuJi-Jim